<ui version="4.0" >
 <class>CategorySelectDialog_base</class>
 <widget class="QWidget" name="CategorySelectDialog_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Categories</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="AutoCheckTreeWidget" name="mCategories" >
     <property name="columnCount" >
      <number>1</number>
     </property>
     <column>
      <property name="text" >
       <string>Category</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
       <spacer>
         <property name="orientation" >
           <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
           <size>
             <width>20</width>
             <height>20</height>
           </size>
         </property>
       </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="mButtonClear" >
       <property name="text" >
        <string>&amp;Clear Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="mButtonEdit" >
       <property name="text" >
        <string>&amp;Edit Categories...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutoCheckTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>autochecktreewidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
    <class>KPushButton</class>
    <extends>QPushButton</extends>
    <header>KPushButton</header>
    <container>0</container>
    <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mCategories</tabstop>
  <tabstop>mButtonClear</tabstop>
  <tabstop>mButtonEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
